doc/source/cookbook.rst

```diff
@@ -1226,6 +1226,17 @@ Computation
 Correlation
 ***********
 
+Often it's useful to obtain the lower (or upper) triangular form of a correlation matrix calculated from `DataFrame.corr`.  This can be achieved by passing a boolean mask to `where` as follows:
+
+.. ipython:: python
+
+    df = pd.DataFrame(np.random.random(size=(100, 5)))
```

Comment:
Could you make the size a lot smaller (like (4, 4) max)? `corr_mat` would be huge otherwise.

Reply:
I think `corr_mat` is only 5 x 5 here

Reply:
Sure, will do